v1_21_4/src/main/java/me/aleksilassila/litematica/printer/v1_21_4/guides/placement/TorchGuide.java

```diff
@@ -28,4 +34,19 @@ protected List<Direction> getPossibleSides() {
     protected Optional<Block> getRequiredItemAsBlock(ClientPlayerEntity player) {
         return Optional.of(state.targetState.getBlock());
     }
+
+    @Override
+    protected boolean statesEqual(BlockState state1, BlockState state2) {
```

Comment:
This can be removed and instead make TorchGuide extend PropertySpecificGuesserGuide